docs/en/stack/ml/nlp/ml-nlp-ner-example.asciidoc

```diff
@@ -0,0 +1,296 @@
+[[ml-nlp-ner-example]]
+= How to deploy named entity recognition (NER)
+
+++++
+<titleabbrev>Named entity recognition</titleabbrev>
+++++
+:keywords: {ml-init}, {stack}, {nlp}
+
+This page demonstrates how to deploy a 
+<<ml-nlp-ner,named entity recognition (NER)>> model in {es}, test the model, and 
+add it to an {infer} ingest pipeline. The model that is used in the example is 
+publically available on https://huggingface.co/[HuggingFace].
+
+
+[discrete]
+[[ex-ner-requirements]]
+== Requirements
+
+include::ml-nlp-shared.asciidoc[tag=nlp-requirements]
+
+
+[discrete]
+[[ex-ner-deploy]]
+== Deploying a NER model
```

Comment:
Not mandatory, but I think it works just as well with the shorter "Deploy" (Ditto for the subsequent sections)
```suggestion
== Deploy a NER model
```